src/index.html

```diff
@@ -5,8 +5,409 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link 
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="style-header">
+      <a href="#Home.html">
+        <img 
+          class="logo" 
+          src="images/logo.png"
+          alt="MoyoLogo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item active-item">
```

Comment:
The problem with linter was with active-item class
instead it you can make pseudo element on nav__link--is-active
```
 &--is-active {
        color: $color-accent-blue;
        position: relative;

        &::after {
          content: "";
          height: 4px;
          width: 100%;
          border-radius: 8px;
          background-color: $color-accent-blue;
          position: absolute;
          bottom: 0;
        }
      }
```

Reply:
and styles work with is-active class, if it set below nav_link in styles